### ISSUE TITLE
[Bug]: Invisible blocks, breaking them crashes game

### ISSUE THREAD

**Issue by TheSilentHawke** [cavecomputing/SWEM-Beta#80]
### What happened?

I was making jumps to practice on, and was flying around in creative and a couple blocks above the edge of the jump there was an invisible block, like a barrier. When I finally found it and broke it, it crashed the game.

### What should happen?

There shouldn't be any invisible blocks

### Mods

geckolib-forge-1.16.5-3.0.35
swem-1.16.5-0.1.1.1-b

### SWEM Version

0.1.1.1-b

### Singleplayer or Multiplayer

- [X] Singleplayer
- [ ] Multiplayer
- [ ] Other (Please add a comment to the issue where this is happening)

### OS

Windows

### Relevant log output

```shell
It did crash, but when I was trying to copy/paste the crash report, it closed out. I will try to replicate it and paste it in the comments if I can't edit the report.
```

**Comment by TheSilentHawke** [comment]
[crash-2021-11-04_17.53.41-client.txt](https://github.com/cavecomputing/SWEM-Beta/files/7479339/crash-2021-11-04_17.53.41-client.txt
![2021-11-04_17 53 34](https://user-images.githubusercontent.com/93680650/140431459-8e1aad20-a798-41d1-a41f-5707aa5387e1.png)
)

**Comment by MagnusHJensen** [comment]
Yeah, that is a thing.

That small block there, is the "brains" of the jump system.
We might need to look into a better way, since a lot of people go the approach using no standards, and then decorating the sides themselves.

I will look into the crash if it's broken, it should just remove the entire jump if it gets destroyed.

**Comment by TheSilentHawke** [comment]
On Fri, Nov 5, 2021 at 3:34 AM Magnus Jensen ***@***.***>
wrote:

> Yeah, that is a thing.
>
> That small block there, is the "brains" of the jump system.
> We might need to look into a better way, since a lot of people go the
> approach using no standards, and then decorating the sides themselves.
>
> I will look into the crash if it's broken, it should just remove the
> entire jump if it gets destroyed.
>

Ohh okay! Yeah, after the game crashes and you load back in, the jump is
destroyed

>

**Comment by MagnusHJensen** [comment]
The crash is fixed, it will destroy the jump if you destroy the block, but no reason for it to crash ;)

This will be fixed with version: `0.1.1.2-b` and up.

**Comment by TheSilentHawke** [comment]
Awesome, thank you!
